wp-includes/widgets/class-wp-widget-media-video.php

```diff
@@ -70,6 +70,12 @@ public function get_instance_schema() {
 					'default' => '',
 					'sanitize_callback' => 'wp_kses_post',
 				),
+				'poster' => array(
```

Comment:
And this https://codex.wordpress.org/Function_Reference/wp_video_shortcode#Attributes clarifies it further.  _goes for 2nd cup of ☕️ _